### DISCUSSION TITLE
I had completely disappointed

### DISCUSSION THREAD

**Discussion by Tallithegirl** [simple-login/app#1591] [Kind words ❤️]
To begin with, why is the forum on github? I had to register, but not the point.

Completely disappointed with the service. Earlier I was glad that finally there was a good service for creating Email aliases. But it didn't work out that way. A couple of months after the registration developers have removed two free domains (left 2) making the service completely useless.

I really hope that this is just a glitch and error, but considering the number of messages about it in social networks and the huge amount of abuse towards the developers in the thematic channels I am beginning to doubt that this is just a bug.

Be that as it may hope that everything will return as it was.

**Comment by nguyenkims** [comment]
This is duplicated to this discussion https://github.com/simple-login/app/discussions/1589, please continue the discussion there.
